plugins/dbhistory/saveSong.js

```diff
@@ -39,7 +40,13 @@ var saveSong = {
             var req = http.request(options, function(resp) {
                 console.log('saveSong response code: '+resp.statusCode);
                 resp.on('data', function(chunk) {
-                    request.sendChat(chunk);
+                    if (resp.statusCode == 403 && saveSong.config.verboseErrors) {
+                        request.sendChat('The song vote history was not saved to the database. It looks as though your api key needs to be updated in your GSBOT config.')
+                    }
+                    else if (saveSong.config.verboseErrors) {
+                        request.sendChat('There was an error code '+resp.statusCode+' encountered while saving the song vote history.')
+                        console.log('BODY: ' + chunk);
+                    }
                     console.log('HEADERS: '+JSON.stringify(resp.headers));
                 })
 
```

Comment:
I forgot to mention that on your previous commit, you should check for req.on('error', function(errCallback) {});, otherwise, you might get a exception on timeout and kill the bot.